pkg/model/dev.go

```diff
@@ -120,6 +120,8 @@ type BuildInfo struct {
 	CacheFrom  []string    `yaml:"cache_from,omitempty"`
 	Target     string      `yaml:"target,omitempty"`
 	Args       Environment `yaml:"args,omitempty"`
+	Image      string      `yaml:"image,omitempty"`
+	Secrets    []string    `yaml:"secrets,omitempty"`
```

Comment:
why are we adding secrets?

Reply:
because is also an option at the cli command, we are not using this then?